platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetScoreCalculator.java

```diff
@@ -78,40 +81,40 @@ public class TipsetScoreCalculator {
     /**
      * The maximum possible advancement score for an event.
      */
-    private final long maximumPossibleScore;
+    private final long maximumPossibleAdvancementWeight;
 
     /**
-     * The previous tipset score.
+     * The previous tipset advancement score.
```

Comment:
You've changed some of the terminology to `weight`, but usage of `score` still dominates in comments and variable names

I'm not saying all `score`s need removing, but maybe skim through the existing wording to see if you think any should be changed

Reply:
updated comments to use the term "weight"